client/src/App.js

```diff
@@ -1,30 +1,103 @@
-import React, {useEffect} from 'react';
+import React, {useEffect, useState} from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { createMuiTheme, makeStyles } from '@material-ui/core/styles';
 import {useSelector, useDispatch} from 'react-redux'
 import './App.css';
 import allActions from './actions'
 import Album from './component/Album'
-
+import Background from './component/Background';
+import Category from './component/Category';
+import Grid from './component/Grid';
+import {Copyright} from './component/Copyright';
+import {NAVER_COLOR, BLUE_COLOR} from './models/colors';
+import {DATA} from './component/Data';
 
 const App = () => {
   const counter = useSelector(state => state.counter)
   const currentUser = useSelector(state => state.currentUser)
-
+  const currentCategory = useSelector(state => state.currentCategory)
   const dispatch = useDispatch()
-
+  
   const user = {name: "Rei"}
+  const classes = useStyles();
+  const [category, setCategory] = useState('경제');
+  const datas = DATA;
 
   useEffect(() => {
     dispatch(allActions.userActions.setUser(user))
+    dispatch(allActions.categoryActions.setCategory(category))
+    let items = [];
+    datas.forEach((doc) => {
+      const cate = doc.cate;
+      const datetime = doc.datetime;
+      const title = doc.title;
+      if ((category == cate)) {
```

Comment:
(()) 로 사용하였는데 아래와 같이 괄호 한개로 수정 부탁드립니다.
```suggestion
      if (category == cate) {
```